backend/mlarchive/archive/views.py

```diff
@@ -940,7 +946,7 @@ def get_context_data(self, **kwargs):
         form = DateForm(self.request.GET)
         
         # if no date submitted default to last month
-        if not self.request.GET:
+        if 'start_date' not in self.request.GET and 'end_date' not in self.request.GET:
```

Comment:
Based on the form validation for that report page, it is not possible to get just one date, both are required. Do you want me to remove this requirement? Also, note that the end date is non-inclusive, which was clean to code and aligns with Python ranges, slices, etc. Do you think this would be confusing to users? Would an inclusive end date be better?

Reply:
Yes, people would expect inclusive endpoints. You can use timedelta to nudge what you send to slices/ranges.